index.bs

```diff
@@ -168,6 +169,12 @@ type: dfn
       which was initiated by a secure context but is downloaded over an
       insecure connection.
     </dd>
+
+    <dt><dfn export>non loopback IP address</dfn></dt>
+    <dd>
+      A non loopback <a>IP address</a> is an address that is not contained in
+      CIDR <i>127.0.0.0/8</i> or <i>::1/128</i>.
```

Comment:
> Now `localhost` itself we only bind to `::1` and `127.0.0.1`, not `127.0.0.2` or some such. Should this really cast a wider net? Has there been discussion about this already somewhere? I don't see an issue reference.

I chose to do this to be in accordance with [the secure-contexts spec](https://www.w3.org/TR/secure-contexts/#is-origin-trustworthy), where we consider the whole ranges to be "Potentially Trustworthy". Since that, I think it wouldn't make sense to reduce the scope here, since it would be in conflict with the second bullet point of [mixed-content spec](https://www.w3.org/TR/mixed-content/#should-block-fetch).

What do you think? (I would have liked to link issue #17  to this PR, but it appears I do not have the correct rights to do so...).